bin/node/runtime/src/lib.rs

```diff
@@ -500,7 +500,8 @@ impl frame_system::offchain::CreateTransaction<Runtime, UncheckedExtrinsic> for
 		account: AccountId,
 		index: Index,
 	) -> Option<(Call, <UncheckedExtrinsic as traits::Extrinsic>::SignaturePayload)> {
-		let period = 1 << 8;
+		// make sure it does not exceed `BlockHashCount`
```

Comment:
How the line below that makes it certain?

Reply:
We could do:
```
BlockHashCount::get().checked_next_power_of_two().map(|c| c / 2).unwrap_or(2)
```